rest-api-spec/src/main/resources/rest-api-spec/test/search/160_exists_query.yml

```diff
@@ -562,22 +562,6 @@ setup:
 
   - match: {hits.total: 4}
 
----
```

Comment:
is this the only yaml test case that will be affected? Just wanted to make sure we have coverage in 7.x to bring this with compatible api

Reply:
Yes this is the only one, we had very light test coverage for this capability.